Source/WebKit/Shared/Extensions/WebExtensionDynamicScripts.serialization.in

```diff
@@ -44,4 +44,21 @@ struct WebKit::WebExtensionScriptInjectionResultParameters {
     std::optional<WebKit::WebExtensionFrameIdentifier> frameID;
 }
 
+struct WebKit::WebExtensionRegisteredScriptParameters {
+
+    std::optional<Vector<String>> css;
+    std::optional<Vector<String>> js;
+
+    String identifier;
+    String runAt;
```

Comment:
This should use `WebCore::UserScriptInjectionTime` or `WebExtension::InjectionTime` enum (likely the WebExtension enum). We want to avoid leaking the API naming, and parsing, up into the UI implementation where possible.